src/Microsoft.Tye.Hosting/TyeHost.cs

```diff
@@ -298,6 +306,31 @@ private async Task StopAsync()
             _processor = null;
         }
 
+        private void OpenDashboard(string url)
+        {
+            try
+            {
+                //still not addressed https://github.com/dotnet/corefx/issues/10361
```

Comment:
It sounds like from here: https://github.com/dotnet/runtime/issues/17938#issuecomment-380770589 that this does work now if you use `UseShellExecute = true`. I don't really mind the code here because I think it's super clear what's happening longform.

Either way could you change this comment to be less negative?

Reply:
I have had random NoOp using ShellExec on other command line apps  [dotnet azure deploy]. This works 100%. Removing comment.